tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -403,11 +423,134 @@ def test_model_monitoring_voting_ensemble(self):
                         <= actual[feature]["mean"]
                         <= actual[feature]["max"]
                     )
-                # overall dritf analysis (details dashboard)
+                # overall drift analysis (details dashboard)
                 for measure in measures:
                     assert measure in drift_measures
                     assert type(drift_measures[measure]) == float
 
+    @pytest.mark.timeout(350)
```

Comment:
what is the average time it has to take? if takes more than 350 you consider that as failed?

Reply:
I have updated this test so now it should not take more than 200 sec. Basically I removed the part of testing model monitoring batch because we also test it in `test_model_monitoring_voting_ensemble`. In the bottom line, this test is now shorter.